### PR TITLE
Add: Trados.TradosStudio2021 16.1.4276 after company rename

### PR DESCRIPTION
**Pull request by wyrdfish**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/63994)

### CODE REVIEW

File: manifests/t/Trados/TradosStudio2021/16.1.4276/Trados.TradosStudio2021.locale.en-US.yaml [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
+PackageIdentifier: Trados.TradosStudio2021
 PackageVersion: 16.1.4276
 PackageLocale: en-US
-Publisher: Rws
+Publisher: SDL Limited (a part of the RWS Holdings Plc group)
 PublisherUrl: https://www.rws.com/
 PublisherSupportUrl: https://www.rws.com/support/
-#PrivacyUrl: 
-#Author: 
 PackageName: Trados Studio 2021
```
Comment (ItzLevvie) [inline manifests/t/Trados/TradosStudio2021/16.1.4276/Trados.TradosStudio2021.locale.en-US.yaml:10]:
```suggestion
PackageName: SDL Trados Studio 2021 SR1
```
This must match Control Panel for `winget list` mapping.

File: manifests/t/Trados/TradosStudio2021/16.1.4276/Trados.TradosStudio2021.locale.en-US.yaml [hunk 2]
```diff
@@ -1,18 +1,16 @@
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
-PackageIdentifier: Rws.TradosStudio
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Trados.TradosStudio2021
 PackageVersion: 16.1.4276
 PackageLocale: en-US
-Publisher: Rws
+Publisher: SDL Limited (a part of the RWS Holdings Plc group)
```
Comment (ItzLevvie) [inline manifests/t/Trados/TradosStudio2021/16.1.4276/Trados.TradosStudio2021.locale.en-US.yaml:7]:
```suggestion
Publisher: SDL
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/t/Trados/TradosStudio2021/16.1.4276/Trados.TradosStudio2021.locale.en-US.yaml (hunk 1 above)
Comment (wyrdfish) [reply manifests/t/Trados/TradosStudio2021/16.1.4276/Trados.TradosStudio2021.locale.en-US.yaml]: Matched to ARP

File: manifests/t/Trados/TradosStudio2021/16.1.4276/Trados.TradosStudio2021.locale.en-US.yaml (hunk 2 above)
Comment (wyrdfish) [reply manifests/t/Trados/TradosStudio2021/16.1.4276/Trados.TradosStudio2021.locale.en-US.yaml]: Matched to ARP

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/t/Trados/TradosStudio2021/16.1.4276/Trados.TradosStudio2021.locale.en-US.yaml [hunk 3]
```diff
@@ -1,18 +1,16 @@
...
-#PrivacyUrl: 
-#Author: 
-PackageName: Trados Studio 2021
+PackageName: SDL Trados Studio 2021 SR1
 PackageUrl: https://www.rws.com/translation/software/trados-studio/
-License: Copyright © 2011–2021 SDL as part of the RWS Holdings Plc group of companies ("RWS Group")
-#LicenseUrl: 
-Copyright: Copyright © 2011–2021 SDL as part of the RWS Holdings Plc group of companies ("RWS Group")
-#CopyrightUrl: 
+License: Copyright © 2011-2022 SDL Limited as part of the RWS Holdings Plc group of companies ("RWS Group")
```
[the remark this answers is not in the document]
Comment (wyrdfish) [reply manifests/t/Trados/TradosStudio2021/16.1.4276/Trados.TradosStudio2021.locale.en-US.yaml]: added Proprietary

**Pull request closed without merging**